From 69e3a37715600761358c58d71a777d292cb429dd Mon Sep 17 00:00:00 2001
From: Yasir-Khan <yasir_khan@mentor.com>
Date: Mon, 31 Oct 2016 16:45:46 +0500
Subject: [PATCH] mali_linux_trace: fix typo in stringify macro

When mali kernel module is built with tracing enabled, we get
compilation error due to a typo in stringify macro.

Upstream-status: Patch sent to meta-xilinx

Signed-off-by: Yasir-Khan <yasir_khan@mentor.com>
---
 driver/src/devicedrv/mali/linux/mali_linux_trace.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/driver/src/devicedrv/mali/linux/mali_linux_trace.h b/driver/src/devicedrv/mali/linux/mali_linux_trace.h
index c6cd2bf..7c27b92 100644
--- a/linux/mali_linux_trace.h
+++ b/linux/mali_linux_trace.h
@@ -18,7 +18,7 @@
 
 #undef  TRACE_SYSTEM
 #define TRACE_SYSTEM mali
-#define TRACE_SYSTEM_STRING __stringfy(TRACE_SYSTEM)
+#define TRACE_SYSTEM_STRING __stringify(TRACE_SYSTEM)
 
 #define TRACE_INCLUDE_PATH .
 #define TRACE_INCLUDE_FILE mali_linux_trace
-- 
2.8.1

